src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationService.kt

```diff
@@ -35,10 +37,19 @@ class CalculationService(
       calcAndExtract(deepCopy(booking), sds40Options.copy(allowSDSEarlyRelease = false))
 
     val latestReleaseDateFromStandardBooking = extractionService.mostRecentSentence(standardWorkingBooking.getAllExtractableSentences(), SentenceCalculation::adjustedDeterminateReleaseDate).sentenceCalculation.adjustedDeterminateReleaseDate
+    val areCalculationsTheSame = areCalculationsTheSame(sds40Result, standardResult)
+    val isLatestReleaseDateBeforeTrancheOne = latestReleaseDateFromStandardBooking.isBefore(trancheOne.trancheCommencementDate)
+
+    log.info(
+      "Return Longest Possible Sentences?: {}, Are Calculations the Same?: {}, Is Latest Release Date Before Tranche One?: {}",
+      returnLongestPossibleSentences,
+      areCalculationsTheSame,
+      isLatestReleaseDateBeforeTrancheOne,
+    )
 
     val tranche =
-      if (returnLongestPossibleSentences || areCalculationsTheSame(sds40Result, standardResult) || latestReleaseDateFromStandardBooking.isBefore(trancheOne.trancheCommencementDate)) {
-        SDSEarlyReleaseTranche.TRANCHE_0
```

Comment:
Just some code tidy here, removed line 41 as it wasnt doing anything, replaced with a  comment